### PR TITLE
Add section 'Test Templates' to the Style Guide

### PR DESCRIPTION
**Pull request by Mko13**
Changes are made, and tested using Docusaurus ~~2.2, since upgrade PR is not merged yet.~~ v3.4.0.

### CODE REVIEW

File: website/docs/style_guide.md [hunk 1]
```diff
@@ -1808,3 +1808,202 @@ Attribute Variables
...
+    a            Hello Word!
+    aaaaaaaaa    @{TEST TAGS}
+    aaaa         ${NONE}
+
+Test Case With Template B
+    [Template]    Template B
+    bbbbbbbbbbb    456          ${1.5}
+    bbbbbb         ${EMPTY}     anything
+    bbb            something    7899999999999
+    bbbbbbbb       ${2}         ${EMPTY}
```
Comment (guidodemmenie) [inline website/docs/style_guide.md:1940]:
```suggestion
Test Case With Template B
    [Template]    Template B
    bbbbbbbbbbbbb    456          ${1.5}
    bbbbbbbb         ${EMPTY}     anything
    bbb              something    7899999999999
    bbbbbbbbb        ${2}         ${EMPTY}
```

File: website/docs/style_guide.md [hunk 2]
```diff
@@ -1808,3 +1808,202 @@ Attribute Variables
...
+    [Tags]    B
+    [Arguments]    ${arg_one}    ${arg_two}    ${arg_three}
+    Log Many    ${arg_one}    ${arg_two}    ${arg_three}
+```
+
+  </TabItem>
+</Tabs>
+
+In **Example 1**, section name `*** Test Cases ***` and `COLUMN1` are separated by 4 spaces, as well as following
+columns. The distance is measured between the longest item in the given column and the start of the folowing column.
```
[the remark this answers is not in the document]
Comment (Mko13) [reply website/docs/style_guide.md]: Thanks, fixed!

File: website/docs/style_guide.md [hunk 3]
```diff
@@ -1808,3 +1808,202 @@ Attribute Variables
...
+Documentation    Eeach test case using different template.
+
+
+*** Test Cases ***
+Test Case With Template A
+    [Template]    Template A
+    aa           123
+    a            Hello Word!
+    aaaaaaaaa    @{TEST TAGS}
+    aaaa         ${NONE}
```
[the remark this answers is not in the document]
Comment (Mko13) [reply website/docs/style_guide.md]: Updated.

File: website/docs/style_guide.md (hunk 1 above)
Comment (Mko13) [reply website/docs/style_guide.md]: Updated.

**Pull request merged**